src/ducks/files/rename.js

```diff
@@ -0,0 +1,73 @@
+/* global cozy */
+
+import { getFiles } from './files'
+import { META_DEFAULTS as meta, extractFileAttributes } from '../../actions'
+
+// constants
+
+const START_RENAMING = 'START_RENAMING'
+const ABORT_RENAMING = 'ABORT_RENAMING'
+export const RENAME_SUCCESS = 'RENAME_SUCCESS'
```

Comment:
Regarding  the ducks architecture, there is no need to export a constant as the ducks should contain everything necessary and sufficient to perform redux's tasks.